interfaces/issue-data.ts

```diff
@@ -50,7 +50,14 @@ export interface IssueData {
   working: string[];
   fundedAt?: Date;
   benefactors?: fundingBenefactor[];
-  disputes?: {address: string; weight: number}[];
+  disputes?: disputes[];
+}
+
+export interface disputes {
```

Comment:
This interface needs to be defined _before_ being used (for readability sake). Also, start the naming with a capital letter, `Disputes` instead of a lower cased one